### ISSUE TITLE
Code produces errors

### ISSUE THREAD

**Issue by datarelish** [MicrosoftLearning/mslearn-dp100#11]
The workspace is not imported with the following error:

TypeError: _get_ambient_new() takes 1 positional argument but 2 were given. 

It is not possible to call Workspace class and save to variable ws, MSLearning Path module 01.

Using the following code also fails to load the workspace. 

 from azureml.core import Workspace
    
 subscription_id = "<my-subscription-id>"
 resource_group = "<my-resource-group>"
 workspace_name = "<my-workspace-name>"
    
 ws = Workspace(subscription_id = subscription_id, resource_group = resource_group, workspace_name = workspace_name)
 ws.write_config()

**Comment by GraemeMalcolm** [comment]
It looks like this was a bug introduced in an update to the Azure ML SDK. It looks like it's been resolved in 1.20.0 (which is currently the version installed on Compute Instances by default)

**Comment by datarelish** [comment]
Hello Graeme, thank you for looking into it. I worked out that the timing of the VM was out of sync and once I synced the time on the VM, recreated the Azure artefacts such as the resource group, workspace, compute instance and the compute cluster, the call to the Azure Workspace went through ok. The code fails at the Workspace call, so I did some digging and this seemed to work. I hope that helps someone else. Thanks, Kind Regards, Jen
